build.gradle

```diff
@@ -334,60 +326,43 @@ dependencies {
-    implementation group: 'org.springframework', name: 'spring-jcl'
-    implementation group: 'org.springframework', name: 'spring-jdbc'
-    implementation group: 'org.springframework', name: 'spring-orm'
-    implementation group: 'org.springframework', name: 'spring-tx'
-    implementation group: 'org.springframework', name: 'spring-web'
-    implementation group: 'org.springframework', name: 'spring-webmvc'
 
     implementation group: 'io.github.openfeign.form', name: 'feign-form', version: versions.feign
     implementation group: 'io.github.openfeign.form', name: 'feign-form-spring', version: versions.feign
-    implementation "io.github.openfeign:feign-httpclient:11.10"
+    implementation "io.github.openfeign:feign-httpclient:13.5"
     implementation group: 'jakarta.servlet', name: 'jakarta.servlet-api', version: '6.1.0'
     implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: versions.log4j
     implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: versions.log4j
     implementation group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: versions.log4j
 
     implementation group: 'com.github.ben-manes.caffeine', name: 'caffeine', version: '3.1.8'
-    implementation group: 'com.sun.xml.bind', name: 'jaxb-osgi', version: '4.0.5'
 
     implementation group: 'com.github.hmcts', name: 'service-auth-provider-java-client', version: versions.reformS2sClient
 
     implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
-    implementation group: 'com.fasterxml.jackson', name: 'jackson-bom', version: '2.18.0', ext: 'pom'
+    implementation group: 'com.fasterxml.jackson', name: 'jackson-bom', version: '2.18.2', ext: 'pom'
     implementation group: 'io.jsonwebtoken', name: 'jjwt', version: '0.9.1'
-    implementation group: 'javax.inject', name: 'javax.inject', version: '1'
-    implementation 'com.github.hmcts:idam-java-client:2.1.1'
+    implementation group: 'jakarta.inject', name: 'jakarta.inject-api', version: '2.0.1'
+    implementation 'com.github.hmcts:idam-java-client:3.0.3'
     implementation "org.springframework.boot:spring-boot-starter-oauth2-client"
     implementation "org.springframework.boot:spring-boot-starter-oauth2-resource-server"
-    implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.41.2'
-    implementation group: 'commons-lang', name: 'commons-lang', version: '2.6'
+    implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.47'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.17.0'
+    implementation group: 'org.apache.commons', name: 'commons-text', version: '1.12.0'
     implementation 'org.springframework.boot:spring-boot-starter-validation'
-    //Fix for CVE-2021-29425
-    implementation 'commons-io:commons-io:2.8.0'
```

Comment:
`./gradlew dependencyInsight --no-single-path --rerun --configuration runtimeClasspath --dependency commons-io`
reveals that commons-io 2.11 is being pulled in transitively. please upgrade to 2.17.0 (or newer)